proto/decentraland/social/friendships/friendships.proto

```diff
@@ -78,6 +78,12 @@ message UpdateFriendshipPayload {
   optional Payload auth_token = 2;
 }
 
+message MutualFriendsPayload {
+  User other_user = 1;
```

Comment:
For the other messages, we've referred to the 'other_user' just as 'user' as we never send the address of the auth user. Maybe we can keep that to avoid confusion 🤔